### DISCUSSION TITLE
VLC Buffering Reduction

### DISCUSSION THREAD

**Discussion by tom76017** [GerryDazoo/Slinger#198] [Show and tell]
I've seen improved screen updates from remote control commands by deactivating the hardware decoding.  The default was set to "automatic" with 5-6 second delays.  Now I'm seeing the results in 3 seconds on external connections.

![image](https://user-images.githubusercontent.com/26171888/198877009-647fe672-a6ee-4429-895e-b73a365d48b1.png)

**Comment by GerryDazoo** [comment]
this also helps. This is the command line arg but you can also change it in preferences
--network-caching=100
